.travis.yml

```diff
@@ -1,16 +1,20 @@
-langauge: ruby
+language: ruby
+cache: bundler
```

Comment:
this made it so when we do a build now it takes 1 min 15 seconds instead of 5+

Reply:
Always a fan of speed